lib/src/gotrue_client.dart

```diff
@@ -216,6 +216,52 @@ class GoTrueClient {
         redirectTo: redirectTo, scopes: scopes, queryParams: queryParams);
   }
 
+  /// Allows signing in with an ID token issued by certain supported providers.
+  /// The [idToken] is verified for validity and a new session is established.
+  ///
+  /// This method is experimental.
+  Future<AuthResponse> signInWithIdToken({
```

Comment:
We can probably use the experimental annotation here! https://api.flutter.dev/flutter/meta/experimental-constant.html
```suggestion
  @Experimental()
  Future<AuthResponse> signInWithIdToken({
```

Reply:
I think you would need to add the `meta` package for this.

Reply:
Done!